### PR TITLE
energy.rakuten.co.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/107686#event-6083777497

<details><summary>Screenshot:</summary>

Снимок экрана 2022-02-16 в 20 03 21

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@ pokemon-gsc.g-takumi.com,1rk.net,e-typing.ne.jp,zukai-kikenbutu.com,gingerweb.jp
 otanew.jp###extra > div.column-inner > div.column-inner-2 > div.plugin-memo:last-of-type
 nlab.itmedia.co.jp##.bannar
 nlab.itmedia.co.jp##.headerbannar
+energy.rakuten.co.jp##.banner--layout--wrapper
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:41]: According to the previous report, there is a leftover `.banner--layout--wrapper` and you block `.banner--layout--wrapper`.
I suggest doing a one common rule `energy.rakuten.co.jp##div[class^="banner--layout"]` will be good for current code or `energy.rakuten.co.jp##div[class^="banner-"]`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:41]: Btw, second rule will be good for other sites with a similar banner.

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
Sorry for late reply. This actually is self-promo.

**Comment by zloyden** [comment]
Oh really? I had an ad on my end, but now it redirects to their subdomain.

**Pull request merged**